lib/Newsletter/ApiDataSanitizer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MailPoet\Newsletter;
+
+class ApiDataSanitizer {
+  /** @var NewsletterHtmlSanitizer */
+  private $htmlSanitizer;
+
+  public function __construct(NewsletterHtmlSanitizer $htmlSanitizer) {
+    $this->htmlSanitizer = $htmlSanitizer;
+  }
+
+  public function sanitizeBody(array $body): array {
```

Comment:
I think might be safer to fix just some whitelisted properties so that we minimize potential issues with data loss.